Lib/wave.py

```diff
@@ -259,6 +259,8 @@ def _read_fmt_chunk(self, chunk):
             self._sampwidth = (sampwidth + 7) // 8
         else:
             raise Error('unknown format: %r' % (wFormatTag,))
+        if self._channels == 0:
```

Comment:
If I am not mistaken, then this attribute was called `_nchannels` in previous versions of Python, i.e. 3.4.x. It might be a good idea to use the `getnchannels()` method to access this information instead, which abstracts the concrete name. This has the advantage of making this patch applicable to previous versions of Python, too.